utils/mixins.py

```diff
@@ -42,7 +42,7 @@ def get_queryset(self) -> QuerySet:
 
 class ProjectsWorkPercentageMixin(ContextMixin):
     def get_context_data(self, **kwargs: Any) -> dict:
-        from users.models import CustomUser
+        from users.models import CustomUser  # pylint: disable=import-outside-toplevel
```

Comment:
Why aren't these imported at toplevel anyway?

Reply:
I didn't want to change it during this update. It might be change as separate chore, but now it his PR :)

Reply:
Ok. Guess we should add this to Jira then?

Reply:
Please add it:)